src/components/footer/Footer.jsx

```diff
@@ -0,0 +1,68 @@
+import React from "react";
+import { Link } from "react-router-dom";
+import "./Footer.css";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faYinYang } from "@fortawesome/free-solid-svg-icons";
+
+import siteLogo from "assets/Images/SiteLogo.png";
+import Button from "components/button/Button";
+
+function Footer() {
+  return (
+    <div className="footer">
+      <div className="footer-follow">
+        <img src={siteLogo} alt="" />
+        <div className="footer-follow-us">
+          <h1>Follow Us</h1>
+          <Button
+            name={<FontAwesomeIcon icon={faYinYang} />}
+            className="facebook"
+            onClick=""
+          />
+          <Button name="G" className="google" onClick="" />
+        </div>
+      </div>
+      <div className="footer-content">
+        <div className="nav-linkss">
+          <ul>
+            <li>
+              <Link to="/Restorants">
```

Comment:
make reusable component

save all links to object 
const footerData = [{id:1, title:'For the restaurants', slug='restaurants'}, {id:1, title:'Press Center', slug='press'}]

and use it like this 
footerData.map(data=><ComponentName data={data}/>)

use slug for <Link to={`/${slug}`}